src/components/countInput/index.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useSearchParams } from 'react-router-dom';
+
+import { ReactComponent as ManIcon } from 'assets/man.svg';
+
+import { setAdults, setChildren, setRooms } from 'store/slices/inputsSlice';
+import Counter from '../counter';
+import { CountersWrapper, DoneButton } from './styled';
+import MainFiltersInput from '../common';
+import { useModal } from '../../hooks/useModal';
```

Comment:
я бы разделял так
относительный импорт лучше избегать, если файлы не лежат в одной и той же папке на одном уровне
```suggestion
// внешние либы
import React, { useState } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { useSearchParams } from 'react-router-dom';

// компоненты
import Counter from '../counter';
import MainFiltersInput from '../common';

// внутренние штуки
import  ManIcon from 'assets/man.svg';
import { setAdults, setChildren, setRooms } from 'store/slices/inputsSlice';
import { useModal } from '../../hooks/useModal';
import { CountersWrapper, DoneButton } from './styled';
```